connectors/sources/jira.py

```diff
@@ -448,6 +451,9 @@ def _decorate_with_access_control(self, document, access_control):
         return document
 
     async def _user_information_list(self, key):
+        self._logger.info(
+            f"Fetching users who have read access to the project/issue key: {key.split('=')[1]}"
```

Comment:
Yes, i think keeping INFO log for just projects and DEBUG for issues is also okay, since there can be a large number of issues present inside the project.